config/routes.rb

```diff
@@ -5,22 +5,30 @@
   # See how all your routes lay out with "rake routes".
 
   # You can have the root of your site routed with "root"
-  root 'home#index'
+  root 'tasks#index'
 
   # Example of regular route:
   #   get 'products/:id' => 'catalog#view'
-  get 'tasks/new' => 'home#new'
-  post 'tasks' => 'home#create'
+  get 'tasks/new' => 'tasks#new'
+  post 'tasks' => 'tasks#create'
+  patch 'tasks' => 'tasks#mark_done'
 
-  get 'tasks/:id' => 'home#show'
-  
-  get '/delete_task/:id' => 'home#confirm_remove'
-  delete '/delete_task/:id' => 'home#delete'
+  get 'tasks/:id' => 'tasks#show'
+  get 'tasks/:id/edit' => 'tasks#edit'
+  patch 'edit_task/:id' => 'tasks#update'
```

Comment:
Great work on the routes. They mostly match the rails pattern. This one, I'd suggest as:

```ruby
patch 'tasks/:id' => 'tasks#update
```

Because the `patch` method distinguishes it from `get` which is used for edit.

Also, you can do the same for delete/destroy below:

```ruby
delete 'tasks/:id' => 'tasks#delete'
```